packages/blaze/dombackend.js

```diff
@@ -9,13 +9,32 @@ if (! $jq)
 
 DOMBackend._$jq = $jq;
 
+
+DOMBackend.getContext = function() {
```

Comment:
Please add documentation to this function. I know many internal functions are not documented but we need better documentation of internals to have more people to easily pick up the development.